core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -726,7 +726,11 @@ private[spark] object SparkConf extends Logging {
     DRIVER_MEMORY_OVERHEAD.key -> Seq(
       AlternateConfig("spark.yarn.driver.memoryOverhead", "2.3")),
     EXECUTOR_MEMORY_OVERHEAD.key -> Seq(
-      AlternateConfig("spark.yarn.executor.memoryOverhead", "2.3"))
+      AlternateConfig("spark.yarn.executor.memoryOverhead", "2.3")),
+    KEYTAB.key -> Seq(
+      AlternateConfig("spark.yarn.keytab", "2.4")),
```

Comment:
This should be 2.5, also below. This change won't go into 2.4 at this point...

Reply:
Thank you for the test! I appreciate it. As the final RC Is not cut, it could be added if you think it should.

Reply:
Okay. I will then point to 2.5.